src/app/screens/Receive/index.tsx

```diff
@@ -191,7 +191,7 @@ function Receive() {
   }
 
   return (
-    <div>
+    <div className="h-full flex flex-col overflow-y-auto no-scrollbar">
```

Comment:
are we save to not want scrollbars?

Reply:
We decided against scrollbars a long time ago. We already do this on most screens.  
If we want to change this we should do it in a new issue. Or is this blocking?

Reply:
ok

Reply:
I think we're currently trying to fit everything into the extension-screen without scrollbars:  
https://github.com/getAlby/lightning-browser-extension/issues/1662